app/models/card.rb

```diff
@@ -20,4 +20,7 @@ def original_translated_text_equal
     def full_downcase(str)
       str.mb_chars.downcase.to_s.squeeze(' ').lstrip
     end
+
+    scope :review_date_card, -> { where('review_date <= ?', Time.now.strftime('%Y-%m-%d')).
```

Comment:
Line is too long. [91/80]<br>Inconsistent indentation detected.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.